### PR TITLE
Fix the pager styles, improve trims.

### PR DESCRIPTION
**Pull request by Jeffiy**
* fix page list display issue
* fix font path issue
* move unimportant scripts to the end of body
* fix if tag name end with ',  ', a tag named '  ' will insert into db
* refactor some code
* unified the url path (delete the last '/')
* delete unnecessary semicolon

### CODE REVIEW

**Comment by Kagamine** [comment]
What's the effect with this change?

**Comment by Jeffiy** [comment]
the page list color is black while the background color is also black

File: src/YuukoBlog/wwwroot/assets/Default/styles/yuuko.css [hunk 1]
```diff
@@ -1604,12 +1604,12 @@ body {
...
 
     .post-nav-page a, .post-nav-page a:active, .post-nav-page a:hover, .post-nav-page a:visited {
-        color: #222 !important;
+        /*color: #222 !important;*/
         text-decoration: none;
     }
 
 .post-nav-page-current {
-    background: #323232 !important;
+    background: #636363 !important;
```
Comment (Kagamine) [inline src/YuukoBlog/wwwroot/assets/Default/styles/yuuko.css]: Why change into this color?

Comment (Jeffiy) [inline src/YuukoBlog/wwwroot/assets/Default/styles/yuuko.css]: Sorry ,I can't upload any image.

> Why change into this color? 

I just change the color of `.post-nav-page-current` and `.post-nav-page-current:hover`

**Comment by Kagamine** [comment]
LGTM

**Pull request merged**